### DISCUSSION TITLE
I can’t get into my old account i removed email from.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#139010] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How can i get back into or delete an old account that i unlinked email, sms receiver i used is no longer available, i didnt save recovery codes. Am i just cooked? Is there at least a way to delete the account?

**Comment by Yash-Bandal** [comment]
Hey @PolarWolf67 
It can be tricky to regain access to an account when the recovery options (email, SMS, or recovery codes) are no longer available, but you're not entirely "cooked" bro
1. Check for Other Linked Accounts:
2. Contact Customer Support
3. Submit a Request for Account Deletion:
4. Check Account Deletion Policies(often 6 months to 2 years)

**Reply by ghost** [reply]
What do you mean other linked accounts?

**Comment by ebndev** [comment]
Hi @PolarWolf67,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
